services/restic_backup.md

```diff
@@ -115,7 +115,7 @@ For mariadb do the following:
     0 5 * * * /home/pulsys/.restic/maria_backup.sh
     ```
 
-## Restore from a backup
+## Retrieve a backup
 
 To restore the latest usable postgresql backup from restic, run the following commands:
```

Comment:
It's a little confusing that the header is "Retrieve a backup" and then the first sentence is "To restore..."

Reply:
Agreed, found two places it was doing that and updated them.